test/libsolidity/syntaxTests/functionTypes/selectors.sol

```diff
@@ -0,0 +1,26 @@
+contract B {
```

Comment:
This file does not trigger an error when run through `solc --ir` (and I think this is also tested for all syntax tests)

Reply:
Do we run IR generator on all syntax tests? That's good to know.

Reply:
> Do we run IR generator on all syntax tests? That's good to know.

Not yet.

Reply:
Just ran the IR on all syntax tests and found one unimplemented feature: https://github.com/ethereum/solidity/issues/11024

Reply:
We did that a few weeks ago already (it used to be more like 9 cases or so), so that issue may be a duplicate...

Reply:
I couldn't find an equivalent issue. We probably missed that one. this was the CI job https://app.circleci.com/pipelines/github/ethereum/solidity/13394/workflows/42f7aa23-443b-4847-be5e-b37f8af856b3/jobs/621452/tests#failed-test-2

Reply:
Weird... I do remember that we were wondering if it actually works and that I noted that it's a bit weird even in old codegen... but maybe we still didn't do anything about it or even file an issue :-).